pytest.ini

```diff
@@ -21,3 +21,6 @@ addopts =
 markers =
     only_asyncio: marks tests as only enabled when --reactor=asyncio is passed
     only_not_asyncio: marks tests as only enabled when --reactor=asyncio is not passed
+filterwarnings =
+    ignore:scrapy.downloadermiddlewares.decompression is deprecated
+    ignore:Module scrapy.utils.reqser is deprecated
```

Comment:
I’m OK with this in these specific cases, but in general it may be best to instead filter out those warnings where we import those modules in tests, because if the warnings came from imports outside tests we would want them.